src/sentry/static/sentry/app/views/settings/components/settingsLayout.jsx

```diff
@@ -17,32 +16,61 @@ class SettingsLayout extends React.Component {
     routes: PropTypes.array,
   };
 
+  state = {
+    navVisible: false,
+  };
+
+  componentWillMount() {
+    this.unlisten = this.props.router.listen(() => {
+      this.toggleNav(false);
+    });
+  }
+  componentWillUnmount() {
```

Comment:
nit: I wish prettier would put a space between this function. Maybe we can eslint for this /cc @billyvg, what do you think?

Reply:
Fixed in 619c649